### PR TITLE
[SYCL][NFC] Make the nd_range error message a bit more informative

### PR DESCRIPTION
**Pull request by v-klochkov**
Signed-off-by: Vyacheslav N Klochkov <vyacheslav.n.klochkov@intel.com>

### CODE REVIEW

File: sycl/source/detail/error_handling/enqueue_kernel.cpp [hunk 1]
```diff
@@ -185,11 +191,22 @@ bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
...
+          std::string GlobalWGSize =
+              std::to_string(NDRDesc.GlobalSize[0]) + ", " +
+              std::to_string(NDRDesc.GlobalSize[1]) + ", " +
+              std::to_string(NDRDesc.GlobalSize[2]);
           std::string message =
               LocalExceedsGlobal
-                  ? "Local workgroup size greater than global range size. "
-                  : "Global_work_size not evenly divisible by "
-                    "local_work_size. ";
+                  ? "Local workgroup size {" + LocalWGSize +
```
Comment (sergey-semenov) [inline sycl/source/detail/error_handling/enqueue_kernel.cpp:204]:
```suggestion
                  ? "Local work-group size {" + LocalWGSize +
```

File: sycl/source/detail/error_handling/enqueue_kernel.cpp [hunk 2]
```diff
@@ -185,11 +191,22 @@ bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
...
+                        "} is greater than global range size {" + GlobalWGSize +
+                        "}"
+                  : "Global work size {" + GlobalWGSize +
+                        "} is not evenly divisible by localgroup size {" +
```
Comment (sergey-semenov) [inline sycl/source/detail/error_handling/enqueue_kernel.cpp:208]:
```suggestion
                        "} is not evenly divisible by local work-group size {" +
```

**Review by sergey-semenov — APPROVED** [review]
LGTM, some minor suggestions for consistency.

**Comment by cperkinsintel** [comment]
Looks like the test ( sycl\test\on-device\basic_tests\reqd_work_group_size.cpp )  need to be updated.  Otherwise, LGTM.

**Comment by v-klochkov** [comment]
> Looks like the test ( sycl\test\on-device\basic_tests\reqd_work_group_size.cpp ) need to be updated. Otherwise, LGTM.

Fixed in https://github.com/intel/llvm/pull/2860/commits/e05a86f5e1e37c41bc9d4783d537c51e8a614ba9. Thank you.

File: sycl/source/detail/error_handling/enqueue_kernel.cpp (hunk 2 above)
Comment (v-klochkov) [reply sycl/source/detail/error_handling/enqueue_kernel.cpp]: Fixed in https://github.com/intel/llvm/pull/2860/commits/e05a86f5e1e37c41bc9d4783d537c51e8a614ba9. Thank you.

**Review by sergey-semenov — APPROVED** [review]

**Pull request merged**